### PR TITLE
feat: add misc options for `yarn`/`curl`/`gatsby`

### PR DESCRIPTION
**Pull request by MengsokunVeng**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: src/zzz.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const completionSpec: Fig.Spec = {
+  name: "zzz",
+  description: "",
```
Comment (fedeci) [inline src/zzz.ts:3]: Is this the correct description?
```suggestion
  description: "A command-line MacOS sleep program (like SleepNow)",
```

File: src/zzz.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+  ],
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for zzz",
+    },
+  ],
+  // Only uncomment if zzz takes an argument
+  // args: {}
```
Comment (fedeci) [inline src/zzz.ts:24]:
```suggestion
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for the PR! There are a couple things to fix before merging this, can you handle those please?

**Comment by MengsokunVeng** [comment]
Sorry for late reply and I noticed that i pushed the tested file right after i cloned the project. Thank you.

**Comment by fedeci** [comment]
@MengsokunVeng  is the zzz file supposed to be deleted?

**Comment by SeparateRecords** [comment]
> @MengsokunVeng is the zzz file supposed to be deleted?

I think this was just a file for testing how completions work and wasn't supposed to be included, in the tutorial we push to create a simple test spec

**Comment by MengsokunVeng** [comment]
Yes, it was just a file to test whether i set up the project correctly or
not.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**